index.html

```diff
@@ -1051,6 +1085,41 @@
                 }
             }
 
+            function dragBacklogEnter(ev) {
+                ev.preventDefault();
+                return true;
+            }
+            function dragBacklogOver(ev) {
+                ev.preventDefault();
+                if (ev.target.classList.contains("drop-target")) {
+                    ev.target.classList.add("drop-target-hover");
+                }
+            }
+            function dragBacklogDrop(ev) {
+                if (ev.target.classList.contains("drop-target")) {
+                    ev.target.classList.remove("drop-target-hover");
+                }
+                ev.preventDefault();
+
+                var bugData = JSON.parse(ev.dataTransfer.getData("text"));
+
+                var targetStatus = "CONFIRMED";
+                var targetResolution = "";
+                var targetPriority = defaultPriority;
+                var targetSeverity = defaultSeverity;
+
+                httpGet("/rest/product/" + bzProduct + "?type=enterable&include_fields=default_milestone", function(response) {
+                    writeBug(bugData.id, bugData.status, targetStatus, targetResolution, targetPriority, targetSeverity);
+                });
+
+            }
+
+            function dragBacklogLeave(ev) {
+                if (ev.target.classList.contains("drop-target")) {
+                    ev.target.classList.remove("drop-target-hover");
+                }
+            }
+
             // Register event handlers
             document.getElementById("textProduct").addEventListener("input", function() {
```

Comment:
Wondering if we can reuse the existing dragCard functions? Quite a bit seems common.